pipelines/test/ocp/aws/pipeline.yaml

```diff
@@ -0,0 +1,228 @@
+      type: string
+    - default: ""
+      description: Additional env for testsuite container separated with spaces (e.g. KUADRANT_CONTROL_PLANE__provider_secret=gcp-credentials KUADRANT_KEYCLOAK__url="https://my-sso.net")
+      name: additional-env
+      type: string
+    # Report Portal related parameters
+    - default: full-cycle-ocp-aws
+      description: Prefix of the launch name saved in report portal (nightly, username, manual, etc.). In case of release candidate testing use kuadrant-v<version>, rhcl-v<version>, or authorino-v<version>
+      name: launch-name
+      type: string
+    - default: ""
+      description: Optional launch description for Report Portal
+      name: launch-description
+      type: string
+    - default: testsuite
+      description: Report Portal Project to store test results (testsuite or nightly-testsuite typically)
+      name: rp-project
+      type: string
+    - default: true
+      description: If set to 'true' upload test results to Report Portal
+      name: upload-results
+      type: string
+    # Cleanup
+    - default: false
+      description: If set to 'true' cleanup will be done
+      name: cleanup
+      type: string
+  workspaces:
+    - name: shared-workspace
+      description: "Workspace to store cluster state files"
+  tasks:
+    - name: provision-ocp-aws
+      params:
+        - name: cluster-name
+          value: $(params.cluster-name)
+        - name: region
+          value: $(params.region)
+        - name: master-size
+          value: $(params.master-size)
+        - name: worker-size
+          value: $(params.worker-size)
+        - name: installer-arch
+          value: $(params.installer-arch)
+      workspaces:
+        - name: shared-workspace
+          workspace: shared-workspace
+      taskRef:
+        kind: Task
+        name: provision-ocp-aws
+    - name: helm-deploy
+      retries: 3
+      params:
+        - name: index-image
+          value: $(params.index-image)
+        - name: channel
+          value: $(params.channel)
+        - name: istio-provider
+          value: $(params.istio-provider)
+        - name: operator-name
+          value: $(params.operator-name)
+        - name: kubeconfig-path
+          value: $(tasks.provision-ocp-aws.results.kubeconfig-path)
+        - name: gateway-crd
+          value: $(params.gateway-crd)
+        - name: keycloak-channel
+          value: $(params.keycloak-channel)
+        - name: additional-helm-flags
+          value: $(params.additional-helm-flags)
+        - name: additional-helm-tools-flags
+          value: $(params.additional-helm-tools-flags)
+      runAfter:
+        - provision-ocp-aws
+      taskRef:
+        kind: Task
+        name: helm-deploy
+      workspaces:
+        - name: shared-workspace
+          workspace: shared-workspace
+    - name: do-custom-updates
+      params:
+        - name: kubeconfig-path
+          value: $(tasks.kubectl-login.results.kubeconfig-path)
```

Comment:
I order to do the custom updates (for now it's just enabling user workload monitoring) you need to be logged into the cluster. This expects `kubectl-login` Task to be present in pipeline so you need to add it there.

I would add/move these two Tasks (`kubectl-login` and `do-custom-updates`) just after the `provision-ocp-aws` Task. So the logic would be:
1) provision the cluster
2) log into it
3) enable the user workload monitoring
4) install Kuadrant

Reply:
I can't see a fix for this being applied either. See here how to use `kubectl-login` Task:
https://github.com/Kuadrant/testsuite-pipelines/blob/main/pipelines/test/osd/pipeline.yaml#L316-L331

Looking into that pipeline myself the `do-custom-updates` Task is there only after the `helm-deploy` Task so you can keep the same ordering in this pipeline as well (so you can keep it you have it now). But you need to add that `kubectl-login` Task into the pipeline!